tests/api2/test_disk_wipe.py

```diff
@@ -24,6 +22,40 @@ def test_disk_wipe_exported_zpool_in_disk_get_unused():
         assert disk['exported_zpool'] is None
 
 
+def test_disk_wipe_partition_clean():
+    """
+    Confirm we clean up around the middle partitions
+    """
+    signal_msg = "ix private data"
+
+    disk = call("disk.get_unused")[0]["name"]
+
+    # Create 1 GiB swap and a data partition
+    call('disk.format', disk, 1)
+    parts = call('disk.list_partitions', disk)
+    # This 'assumes' 512 byte sectors
```

Comment:
Why assume?  Does `int(parts[0]['start'] / parts[0]['start_sector'])` yield the sector size?

Reply:
Sure.  I was being lazy and avoiding the block size command to dd